tokio-trace/src/span.rs

```diff
@@ -4,9 +4,11 @@
 //!
 //! A thread of execution is said to _enter_ a span when it begins executing,
 //! and _exit_ the span when it switches to another context. Spans may be
-//! entered through the [`enter`] method, which enters the target span,
-//! performs a given function (either a closure or a function pointer), exits
-//! the span, and then returns the result.
+//! entered through the [`enter`] and [`enter_scoped`] methods.
+//!
+//! The `enter` method performs a given function (either a closure or a
+//! function pointer), exits the span, and then returns the result; while
+//! `enter_scoped` enters the span and returns a [guard] that will enxit
```

Comment:
"enxit"? Seems to also be missing some text?

Reply:
agh, i think the missing text is due to me screwing up a rebase; will fix